benchmarks/oss_fuzz_benchmark_integration.py

```diff
@@ -0,0 +1,243 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Script for setting up an integration of an OSS-Fuzz benchmark. The script
+will create oss-fuzz.yaml as well as copy the files from OSS-Fuzz to build the
+benchmark."""
+import argparse
+import datetime
+from distutils import spawn
+import os
+import logging
+import sys
+import shutil
+import subprocess
+
+from common import utils
+from common import benchmark_utils
+from common import logs
+from common import new_process
+from common import yaml_utils
+
+
+# pytype: disable=import-error
+# pylint: disable=import-error,wrong-import-position,ungrouped-imports,too-many-arguments
+
+# TODO(metzman): Don't rely on OSS-Fuzz code. We don't want to depend on it
+# because it can easily break use. Especially because:
+# 1. We use private methods
+# 2. The OSS-Fuzz code depends on absolute imports.
+# 3. The OSS-Fuzz code assumes it is run from the OSS-Fuzz directory and can
+# accdidentaly break our repo.
+OSS_FUZZ_DIR = os.path.join(utils.ROOT_DIR, 'third_party', 'oss-fuzz')
+OSS_FUZZ_REPO_PATH = os.path.join(OSS_FUZZ_DIR, 'infra')
+
+
+class BaseRepoManager:
+	"""Base repo manager."""
+
+	def __init__(self, repo_dir):
+	    self.repo_dir = repo_dir
+
+	def git(self, cmd, check_result=False):
+	    """Run a git command.
+
+	    Args:
+	      command: The git command as a list to be run.
+	      check_result: Should an exception be thrown on failed command.
+
+	    Returns:
+	      stderr, stdout, error code.
+	    """
+	    return new_process.execute(['git'] + cmd,
+	                         cwd=self.repo_dir)
+
+
+class BaseBuilderRepo:
```

Comment:
Can you call this `BaseBuilderDockerRepo` I think it's confusing to have two classes with `Repo` in the name where one refers to a docker repo and the other refers to a git repo.